### PR TITLE
feat: add LOC creation to SDK.

### PR DESCRIPTION
**Pull request by gdethier**
logion-network/logion-internal#174

### CODE REVIEW

File: src/logion-chain/LogionLOC.tsx [hunk 1]
```diff
@@ -0,0 +1,25 @@
+import { ApiPromise } from '@polkadot/api';
+import { ExtrinsicSubmissionParameters, signAndSend, Unsubscriber } from './Signature';
+import { UUID } from './UUID';
+
+export interface LOCCreationResult extends ExtrinsicSubmissionParameters {
```
Comment (benoitdevos) [inline src/logion-chain/LogionLOC.tsx:5]: Name `LOCCreationResult` is a bit misleading IMHO. `LOCCreationParameters` sounds better.
```suggestion
export interface LOCCreationParameters extends ExtrinsicSubmissionParameters {
```

File: src/logion-chain/LogionLOC.tsx [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+    api: ApiPromise,
+    locId: UUID,
+}
+
+export function createLoc(parameters: LOCCreationResult): Unsubscriber {
```
Comment (benoitdevos) [inline src/logion-chain/LogionLOC.tsx:10]:
```suggestion
export function createLoc(parameters: LOCCreationParameters): Unsubscriber {
```

**Review by benoitdevos — APPROVED** [review]
Good job, one suggestion to improve naming.

File: src/logion-chain/LogionLOC.tsx (hunk 1 above)
Comment (gdethier) [reply src/logion-chain/LogionLOC.tsx:5]: Indeed.

**Pull request merged**